### ISSUE TITLE
Extension doesn't load

### ISSUE THREAD

**Issue by platon-ivanov** [eweren/vscode-tolgee#7]
Hello! This extension looks amazing! 

I would like to try it out, but unfortunately it seems to not be loading at all. There is no field for it in the `Output` window and when I try to run the command `Tolgee: Change language`, the message `command 'tolgee.changeLanguage' not found`.

What could be the problem?

**Comment by eweren** [comment]
Great that you are trying things out.
Do you have the tolgee cli installed and configured for your project?
I'll try to add some debugging logs to the extension, so that this could help

**Comment by eweren** [comment]
Thanks for your quick answer.
I found an issue that will be resolved in a few minutes with v 0.1.2 (caused by a missmatch of commands and extension name)

It might be possible, that the extension still won't work for your setup, because I heavily relied on the 'push' section of the tolgee config to find the path for the static files. But I think I can also make this work by using the path of the pull command. That's maybe even better.

**Comment by eweren** [comment]
yes I found the real issue 😓 A dependency was in dev-dependencies, so - of course - it was not found in the published extension and thus it could not start correctly.
Wait for v0.1.3 in some minutes. It should work then. I also rewrote the part to use the files under pull instead of the push section.

**Comment by eweren** [comment]
... still issues with that. The `schema.json` is not bundled. I guess I should have tested all this before.
If you want to -> take a look at it by cloning this repo, installing the dependencies, pressing F5 and open a tolgee-using-repo in the extension host to get started.

This is probably just an issue with the `package.json` to not include the `.json` file.
Otherwise I will look into it later today or in the next few days

**Comment by eweren** [comment]
Yes that's possible. We just need to adjust the functional regex for that.

Can you possibly create a minimal repro of your setup so that I can test with that?

**Comment by eweren** [comment]
Okay nice. I think that will help reproduce it.
In my test-repos I always used a `.tolgeerc` file. Maybe there is an issue with reading your config.
I'll look into that in a few days.
Thank you! 🙏

**Comment by eweren** [comment]
Okay 👍
Thinking of the missing languages in the change language command, I think either the config isn't read correctly or the language files could not be located or read (just taking a note for later ;) )

**Comment by eweren** [comment]
I just created a repo that only contains the files from your setup and it works for me.

<img width="1490" alt="Bildschirmfoto 2024-12-29 um 19 01 17" src="https://github.com/user-attachments/assets/fab14175-8918-41f6-8f43-6edca5c1b898" />

The only difference I can spot, is that I am using macos instead of arch linux.
I have added some more logs. Could you look if your languages are detected correctly?

**Comment by eweren** [comment]
@platon-ivanov can you try with the newest 0.1.6? I had the same issue on a windows machine and it looks like the paths were not resolved in the same way, vs code on macos does
